hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -99,6 +99,20 @@
   private static final String MAX_CONSISTENCY_CHECKS_PROP = "hoodie.consistency.check.max_checks";
   private static int DEFAULT_MAX_CONSISTENCY_CHECKS = 7;
 
+  /**
+   * HUDI-858 : There are users who had been directly using RDD APIs and have relied on a behavior in 0.4.x to allow
+   * multiple write operations (upsert/buk-insert/...) to be executed within a single commit.
+   *
+   * Given Hudi commit protocol, these are generally unsafe operations and user need to handle failure scenarios. It
+   * only works with COW table. Hudi 0.5.x had stopped this behavior.
+   *
+   * Given the importance of supporting such cases for the user's migration to 0.5.x, we are proposing a safety flag
+   * (disabled by default) which will allow this old behavior.
+   */
+  private static final String ALLOW_UNSAFE_MULTI_OPERATIONS_PER_COMMIT =
+      "hoodie.allow.unsafe.multi_operations_per_commit";
```

Comment:
lets not mix underscore and dot notation? :)

Reply:
@n3nash was also thinking about adding one of these.. Let's give these a consistent name.prefix.. `_.hoodie.allow.multi.write.on.same.instant` .. beginning with underscore can mean (spark also has similar undoced params.. ) its undoc-ed/hidden and to be used with full understanding..